Adding stub project for the API Design Activity for SDK Training

annelo-msft opened the pull request:
See related PR: https://github.com/Azure/azure-sdk-pr/pull/360

MehaKaushik committed:
Fix for failing long running operations (#14663)

Co-authored-by: Meha Kaushik <mekaushi@microsoft.com>

christothes committed:
Implement ctor accepting a connection string (#14674)

* Implement ctor accepting a connection string

* change version

* formatting

* pr comments

* export api

* connectionstring tests

maririos committed:
[TA] SDK now targets 3.1-preview.2 (#14633)

* new features generated + CI enabled + recordings

* update changelog

Commit:
Azure Monitor Exporter - Add Connection String  (#14621)

pakrym committed:
Add release preparation script (#14680)

bikamani committed:
feat(samples): Twin configurations sample (#14683)

pakrym committed:
Release script fixes (#14686)

JoshLove-msft committed:
Amqp updates (#14681)

rajkumar-rangaraj committed:
Azure Monitor Exporter - Adopt OpenTelemetry .NET beta2 changes (#14687)

pakrym committed:
Add Microsoft.Extensions.Azure.WebJobs package (#14589)

JoshLove-msft committed:
Add SubQueue option and remove locktoken overloads (#14692)

chidozieononiwu committed:
Restructure Changelog Logic (#13783)

* Restructure Changelog Logic

* reset eng/common directory

maririos committed:
move test to individual classes (#14684)

JoshLove-msft committed:
Add service version and remove public ctors for output only types (#14690)

seanmcc-msft committed:
Feature/storage/stream position deadlock bug (#14301)

Commit:
Adding SNI support to ClientCertificateCredential (#14636)

* Adding SNI support to ClientCertificateCredential

* updating API ref

christothes committed:
redame content and some additional samples (#14713)

* redame content and some additional samples

* typos

* add Delete perms

seanmcc-msft committed:
Feature/storage/open write page blob bug (#14716)

Commit:
Update code owners for ADT (#14722)

seanmcc-msft committed:
Preparing for Storage release (#14721)

azure-sdk committed:
Increment version for storage releases (#14725)

strehan1993 committed:
Add Azure AD Only Authentication for Managed Instance (#14717)

* Add support for Managed Instance Azure AD Only Auth.

* Fix folder path.

* Fix sqlManagementClient

* Add res-manager, update the api version to 1.44.1-preview

* Fix assembly file version.

heaths committed:
Add LocalCryptographyClient for Key Vault (#14677)

* Initial LocalCryptographyClient

* Add tests for LocalCryptographyClient

* Fix build break from CI using older SDK

* Resolve PR feedback

* Shorten polling intervals for test playback

Fixes #14557

heaths committed:
Mark tests inconclusive if indexer status times out (#14727)

heaths committed:
Change Key Vault package versions to 4.2.0-beta.1 (#14678)

* Change Key Vault package versions to 4.2.0-beta.1

* Change .Administration to 4.0.0-beta.1

AbelHu committed:
Generate code for package-2020-07-01 for Azure Kubernetes Service (#14541)

* Generate code for package-2020-07-01 for Azure Kubernetes Service

* Upgrade tests for AKS 2020-07-01

isra-fel committed:
[KeyVault] Support new resource: managed HSM (#14569)

* mhsm

* mhsm2

* exclude from test

* mhsm3

* bump up version

* all new api versions

* generate from merged spec

* include tests

Jingshu923 committed:
[DataFactory]Added logLevel/enableReliableLogging to LogStorageSettings (#14597)

* Added logLevel/enableReliableLogging to LogStorageSettings

* Update Tar GZip

wonner committed:
[Synapse] Add test cases for Artifacts and update generator version & rerecord tests (#14575)

* add test cases for artifacts

* solve LRO and add test cases

* update generator version and rerecord tests

* update version

* update changelog

* update readme

* generate export API

* update changelog

* update release date

Co-authored-by: Wan Yang <t-wayang@microsoft.com>

Commit:
[Identity] add RedirectUri option to InteractiveBrowserCredentialOptions (#14720)

* [Identity] add RedirectUri option to InteractiveBrowserCredentialOptions

* updating API reference

kasobol-msft committed:
Update CODEOWNERS (#14759)

From observation. Looks like nesting is winning and @tg-msft is no longer added to our PRs. Fixing that here.

maririos committed:
[TA] Add encoding support and length and offset (#14719)

* add encoding and length and offset

* Pr feedback

* export-api

heaths committed:
Clarify KeyClientBuilderExtension methods params (#14737)

* Clarify KeyClientBuilderExtension methods params

Fixes #14695

* Resolve PR feedback

chidozieononiwu committed:
Retire Net core installer (#14567)

* Specify SDK version in b=global.json

* Retire Dotnet Installer Task

* Retire UseDotNet task

* Remove DownloadPipelineArtifact in analyze stage

kasobol-msft committed:
[Storage] Make OpenWrite work with using keyword (#14758)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

timtay-microsoft committed:
Fix ADT Patch APIs to return Response rather than Response<String> (#14729)

updateComponent and updateDigital twin will never get an HTTP response payload by design, but our custom code mistakenly assumed they would. This fixes the customization code so that they return Response rather than Response<String>

allenjzhang committed:
Update mgmt.yml for Azure Pipelines

kasobol-msft committed:
[Storage] Update changelog. (#14772)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

* update changelog

Commit:
[Identity] Add IdentityModelFactory to facilitate mocking (#14769)

* [Identity] Add IdentityModelFactory to facilitate mocking

* update API spec and changelog

Commit:
refactor sdkversion (#14774)

vinagesh committed:
Updating to latest swagger and regenerating code. (#14762)

JoshLove-msft committed:
Lifecycle updates (#14773)

maririos committed:
[TA] Add flags enum for Opinion mining (#14624)

* add flags enum for OM

* PR feedback

* export-api

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 935 (#14693)

chidozieononiwu committed:
Remove use UseDotNetCore Task (#14771)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 916 (#14616)

pakrym committed:
Support reading existing DataProtection KeyVault keys (#14778)

kerri-lee committed:
Updating README and Samples for Event Grid (#14568)

* Updated README

* Created samples README and sample files

* Small edits, updated system event types

Edited test for CloudEvents publishing raw JSON

Added StorageLifecyclePolicyCompletedEvent to list of system event types

* Updating session records

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 930 (#14628)

JoshLove-msft committed:
Dispose processor in tests (#14784)

christothes committed:
Properly segment StorageEndpointSuffix and CosmosEndpointSuffix (#14754)

christothes committed:
Create backup and restore samples, various other updates (#14756)

* samples - pre-recordings

* use CodeGenModel Usage

* take latest swagger spec and remove hacks

* renaming

* version change

* fix readme nuget version

* remove test auth

* fix tests

* update generator and re-record

* links

* thanks, link validation :)

* readme fixes

* prc

* make errors internal

* keep lowercase readme.md until this merges

* regen

pakrym committed:
Store release notes into Notes custom property of the issue (#14782)

audunn committed:
[NetAppFiles]Anf 6504 net app files bump to 2020 02 01 (#14591)

* Update to 2020-02-01 api verison

* mgmt

* Update test recordings

* Update api version number

Co-authored-by: Audunn Baldvinsson <audunn.baldvinsson@netapp.com>

weshaggard committed:
Remove hard-coded .NET Sdk version for mgmt pipelines (#14799)

Commit:
[Identity] Throw CredentialUnavailableException from credentials not supporting ADFS (#14763)

* [Identity] Throw CredentialUnavailableException from credentials not supporting ADFS

* moving tenantId check to after it's read from settings

* fix check to use local variable

ShivangiReja committed:
Generate Anomaly Detector for Preview-2 (#14791)

 Generate Anomaly Detector for Preview-2

christothes committed:
Incorporate tables readme feedback (#14795)

MikhailTryakhov committed:
Added inbound security rule. Fixed Netwatchers (#14728)

christothes committed:
Readme and sample tweaks (#14801)

* Readme and sample tweaks

* Update sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

Co-authored-by: Heath Stewart <heaths@outlook.com>

* Update sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

Co-authored-by: Heath Stewart <heaths@outlook.com>

* Update sdk/keyvault/Azure.Security.KeyVault.Administration/readme.md

Co-authored-by: Heath Stewart <heaths@outlook.com>

Co-authored-by: Heath Stewart <heaths@outlook.com>

maririos committed:
add bingEntitySearchApiId (#14805)

heaths committed:
Change optional Guid name to Guid? (#14827)

* Fix Key Vault Administration README

* Change optional Guid name to Guid?

Fixes #14823

weshaggard committed:
Change prerelease versioning (#14780)

- Use alpha instead of dev
- Use beta instead of preview

maririos committed:
[TA] Add PII back (#14794)

* all pii code

* tests

* recordings

* samples

* pr feedback

* add mocking

* PR feedback

kerri-lee committed:
Fixed samples (#14835)

Parse method doesn't allow trailing commas in JSON

arv100kri committed:
[Azure Cognitive Search][Management plane] Fix typos from swagger (#14787)

* pull in the docs update

* regenerate from master

Co-authored-by: Arvind Krishnaa Jagannathan <arjagann@microsoft.com>

dhawal777 committed:
SDK refresh to support new API version 2020-04-01 (#14708)

* SDK refresh to support new API version 2020-04-01

* databox_resource-manager.txt update

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 946 (#14831)

kerri-lee committed:
Update changelog, csproj, and readme for Event Grid (#14838)

* Update changelog, csproj, and readme

Updating version name to 4.0.0-beta.1

* Fixed changelog date

christothes committed:
Add known issues section to README (#14812)

loic-sharma committed:
Minor Tables readme improvements (#14846)

* Minor Tables readme improvements

* Update readme.md

stowen-msft committed:
[ReleasePR Microsoft.Azure.Management.Automanage] Microsoft.Automanage SDK for 2020 06 30 preview  (#14486)

* Includes Generated Files And Half Of Unit Test Coverage

* tests completed

* updated readme

* updated versions, included AssemblyInfo

* removed unneeded package references

* updated assemblyname to properCasing

* Rename Microsoft.Azure.Management.Automanage.sln to Microsoft.Azure.Management.AutoManage.sln

* casing fixes

* added metadata file

* lineending+

* newline--

* changed hardcoded reference to dotnet coreapp31

* PR feedback, fixed csproj hard coded versions, added required properties to csproj

* fixing ambiguous package name error

* fixed version number

* fix wrong reference in generate

* PRfeedback, fixed incorrect tag references, version numbers, and remove unneeded dotnet version

* pr feedback, fixing missing session records

* renamed missing test

* fixed old test

Co-authored-by: Stephen Owen <stephen.owen@microsoft.com>

ruowan committed:
Create pull_request_assignment.yml (#14748)

* Create pull_request_assignment.yml

* Update pull_request_assignment.yml

jsquire committed:
[Service Bus Client] Connection String SAS Support (#14806)

The focus of these changes is to add support for a precomputed shared
access signature token to be used as part of the connection string.

ShivangiReja committed:
[Anomaly detector] Fix readme links and changelog release date for preview-2 (#14854)

 [Anomaly detector] Fix readme links and changelog release date for preview2

kasobol-msft committed:
[Storage] OpenRead - fail fast if blob doesn't exist. (#14863)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

* fail fast if blob doesn't exist at OpenRead.

chidozieononiwu committed:
Update required net core sdk version (#14804)

maririos committed:
[TA] Samples and documentation for Opinion Mining and Pii (#14829)

* samples, readme + fix version

* update links

* update samples

seanmcc-msft committed:
Fixed bug where Listing Blobs with BlobTraits.Metadata would return B… (#14861)

seanmcc-msft committed:
Fixed bug where DataLakeFileClient.Upload() could not upload read-only files. (#14864)

AlexanderSher committed:
Fix .Net SDK Version (#14875)

maririos committed:
[TA] Update links to point to master (#14871)

* update links to point to master

* english

chidozieononiwu committed:
Switch to using pipeline resource (#14877)

pakrym committed:
Update versions for August release (#14868)

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

rajkumar-rangaraj committed:
Azure Monitor Exporter - Match HTTP URL retrieval  to OpenTelemetry specification (#14834)

bquantump committed:
Add remaining test cases from track one DNS SDK to track two (#14766)

* WIP: Initial CRUD and List test for record and zone

* WIP: Port of cloud issues

* WIP:  OneTimeTearDown and setup working. No double test runs

* WIP: don't make zone twice

* WIP: Commit test

* WIP: Fix changelog and readme

* WIP: Fix yaml

* WIP: Fix yml 2

* WIP: Remove unused function

* Remove .swp file

* Remove SWP file

* Update DNS Track 2 test

* Add Tests Recordings

* Fix Class name

* WIP: Update tests

* WIP: Make exception test clearner

* WIP: Update live test records

* Revert json change

Co-authored-by: bqauntump <bqauntump@DESKTOP-B63NOF4.localdomain>

AlexanderSher committed:
Add support for both .Net Core 2.1 and .Net Core 3.1 (#14884)

* Add support for both .Net Core 2.1 and .Net Core 3.1

* Install runtime instead of SDK

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 955 (#14873)

pakrym committed:
Support calculating and setting release date (#14876)

* Support calculating and setting release date

* FB

* More FB

* Tuesday

pakrym committed:
Add release notes collect script (#14879)

weshaggard committed:
Allow preview label for until they all ship GA (#14881)

chidozieononiwu committed:
Update contributing document (#14870)

jsquire committed:
[Event Hubs Client] Connection String SAS Support (#14858)

The focus of these changes is to add support for a precomputed shared access
signature token to be used as part of the connection string.

jsquire committed:
[Event Hubs Client] Documentation Updates

The focus of these changes is to sync the release documentation for v5.2.0 with the release branch.  The version stamps and references have not been changed, as those will be covered under a dedicated pre-release set of changes.

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 926 (#14610)

pakrym committed:
Move Microsoft.Extensions.Azure into extensions directory (#14715)

sima-zhu committed:
Bump the template version. (#14908)

allenjzhang committed:
Adding additional team member to PR and codeowner for management SDK code path (#14880)

seanmcc-msft committed:
Feature/storage/blob access policy bug (#14872)

azure-sdk committed:
Increment version for core releases (#14907)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 823 (#14656)

chidozieononiwu committed:
Remove reminants of dotnet core versions (#14783)

jsquire committed:
[Azure.Core TestFramework] Fix Required Optional Environment Variable (#14906)

The focus of these changes is to bypass the attempt to santitize an optional
environment variable when no value was present.  Previously, if the
variable was not read, sanitizing would result in an exception.

JoshLove-msft committed:
Fix bug when sanitizing empty body (#14885)

seanmcc-msft committed:
Missed line in DataLakeFileClient.Upload() readonly fix (#14910)

Commit:
[Identity] Fix environment variable mismatch for live CI tests (#14865)

* Identity fix environment variable missmatch for live CI tests

* updating container image for live tests

* updating test container image

mitchdenny committed:
Add release note for new AddConfigurationClient extension. (#14903)

* Add release note for new AddConfigurationClient extension.

* Bump library version.

isra-fel committed:
Update Microsoft.Azure.Management.KeyVault.csproj (#14898)

pakrym committed:
Support WebJobs configuration fallback logic (#14825)

pakrym committed:
Update Azure.Core.* versions (#14923)

* Update Azure.Core.* versions

* order

kerri-lee committed:
Event Grid: Updating Readme and Samples (#14889)

* Revising readme and samples based on arch board feedback

chidozieononiwu committed:
Update Langauage Settings File (#14922)

ShivangiReja committed:
Override ContentType header when sending multipart/form-data type request (#14925)

AlexanderSher committed:
Fix #14408: [BUG] ChainedTokenCredential throws an exception when VisualStudioCodeCredential is present (#14882)

* Tests added

* Fix #14408: [BUG] ChainedTokenCredential throws an exception when VisualStudioCodeCredential is present

sima-zhu committed:
Fixed change log (#14916)

AlexanderSher committed:
Fix #13687: Move to latest App Services MSI API version (#14770)

* Fix #13687: Move to latest App Services MSI API version

* Delete unused type

* Move request builders creation into factory methods

* Move maximum of code into IManagedIdentitySource implementations

* Fix formatting

* Update sdk/identity/Azure.Identity/src/ManagedIdentityCredential.cs

Typo fix

Co-authored-by: Charles Lowell <chlowe@microsoft.com>

Co-authored-by: Charles Lowell <chlowe@microsoft.com>

pakrym committed:
Improve integration of configuration based clients with other features (#14859)

* Improve integration of configuration based clients with other features

* Fixes

azure-sdk committed:
Increment package version after release of Azure.Extensions.AspNetCore.DataProtection.Keys (#14932)

MiYanni committed:
New SDK - SchemaRegistry (#13657)

* Generated the SchemaRegistry SDK. Set up project and custom client. Fixed original swagger and made a separate PR for it.

* Update to latest swagger with description fixes.

* Fixed changlog. Regen surface.

* Regen with correct swagger.

* Started adding client methods.

* Update SchemaId to be struct. Regen api.

* Initial attempt of serializer.

* Update swagger version. Regenerate rest client. Update public surface. Created first working test.

* Added happy path tests for each client method. Added recordings.

* Move serialization stuff into the Azure.Data.SchemaRegistry.Avro project. Created project, test project, and tool (to test Avro).

* Modify SchemaProperties to have smaller set of properties. Rerun and rename tests. Update api.

* Still attempting to get Avro library usable functionality. Some reflection-based usage is working. Unfortunately, we're encoding the avro file incorrectly (it includes schema).

* Initial serializer implementation. Only does specific records.

* Fixed issues with specific record schema de/serialization.

* Got GenericRecord working. Needs cleanup.

* Code cleanup begins!

* :-) Cleaned up code for serializers. Wrote proper async implementation.

* Renamed classes to be more accurate. Removal of commented code.

* Moved serializer over to main project. Removed other project and solution.

* Added 1 test for the serializer.

* Update api.

* Added back the separate solution. Named it appropriately.

* Addressing some changes from API view comments.

* Added some negative tests. Updated public surface descriptions.

* In progress writing readmes.

* Fixed position reset in deserialization stream.

* Made constants for wire format lengths/positions.

* Updated property description. Made static field for UTF8 encoding.

* Removed unused Directory.Build.props.

* Removed unused description.

* Continuing readme. Started adding snippets.

* First readme completed.

* Finally! Created readmes and snippets.

* Put credential scope string into a constant.

* Addressing a couple documentation PR comments and removed a code comment.

* Added global suppressions for namespace name.

* Forgot using for memory streams in tests and snippets.

* Fixed bugged diagnostic scope names. Put scope names into constants.

* Updated ci configuration. Removed resources template since resources cannot be provisioned currently.

* Added the other package to CI.

* Updated sample file names to match convention.

* Added missing readme links.

* Fixed the readme snippets so they don't format oddly.

* Updated schema components descriptions.

* Fixed command to acquire packages.

JoshLove-msft committed:
Rename Atom client (#14931)

MiYanni committed:
[SchemaRegistry] Fix broken test recordings (#14935)

* Update tests to use single-line strings. Recording encoding was keeping the /r/n from Windows and breaking everything.

* Ignoring tests since I cannot figure out the encoding issue.

* Added missing readme sections.

* Ignored the other tests as they suffer from the same issue.

* Fix readme link.

* Really... it needs to say that exact phrase.

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 969 (#14928)

annelo-msft committed:
stub project for api design activity

annelo-msft committed:
swagger updates

brandong-ms committed:
Azure SDK updates for Microsoft.Sql (#14844)

* Update clients for Microsoft.Sql and update tests for ImportExport

* Add SessionRecords for new IE APIs

* Undo change done for local testing

* Include Alicia's client changes

* Add missing models

* Update session record for TestImportExistingDatabase

* adding in updated test/session for create database test (includes storageAccountType check)

* Fix the new ImportExportScenarioTests

* trying out another set of generated clients

* switch back generated clients, updated database create test

* reinstate deleted model, add updated test session

* Undo ManagedInstanceOperation model change due to failing pipeline check - will follow up with MI team to do this update separately

* Update sql_resource-manager.txt

* Checkout Generated before re-generate

* Regenerate clients

* Update Generated after deleting old Generated folder

* Adding coverage for new IE scenarios

* Update SDK version and release notes

* Re-record existing IE tests after changing test resource name pattern

Co-authored-by: xaliciayang <ayang@microsoft.com>

christothes commented on sdk/api-learn/Azure.Learn.Computation/src/Azure.Learn.Computation.csproj:
```diff
@@ -0,0 +1,31 @@
+  <PropertyGroup>
+    <Description>API Design Learning Project</Description>
+    <AssemblyTitle></AssemblyTitle>
+    <Version>0.0.1-preview.1</Version>
```
Should we start snapping to the new `beta` moniker`?

christothes commented on sdk/api-learn/Azure.Learn.Computation/tests/ComputationClientTestEnvironment.cs:
```diff
@@ -0,0 +1,15 @@
+    // To learn more about TestEnvironment classes, please see: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/README.md
+    public class ComputationClientTestEnvironment : TestEnvironment
+    {
+        public ComputationClientTestEnvironment() : base("api-learn")
```
should the string we pass to the base ctor match to the service name?

annelo-msft replied:
https://github.com/Azure/azure-sdk-pr/issues/403

christothes reviewed (approved):
👍

kasobol-msft committed:
[Storage] OpenRead/Write test recordings are back. (#14963)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

* record open write tests.

JoshLove-msft committed:
Fix starvation issue with session processor when MaxCallsPerSession > 1 (#14937)

sima-zhu committed:
Bump template to the one not used. (#14964)

pakrym committed:
Unblock Core build by fixing snippet error (#14966)

HarveyLink committed:
Re-Generate EventHubs and re-record (#14946)

* Re-Generate EventHub and re-record

* Update record

hutzleo committed:
new sdk 25.2 (#14948)

HarveyLink committed:
Re-generate test common and AppConfiguration. (#14902)

Tianyun-G committed:
Fix keyvault Readme issues (#14839)

* Fix key vault readme issues

* modify

heaths committed:
Add getcert sample for Key Vault (#14971)

Resolves part of #12742

christothes committed:
set release date in changelog (#14972)

* set release date in changelog

* content

jsquire committed:
[Event Hubs Client] September Release Updates (#14965)

The focus of these changes is to update versions and supporting assets to
reflect the September release of the Event Hubs client packages.

nisha-bhatia committed:
Added CassandraTest (#14886)

azure-sdk committed:
Increment package version after release of Azure.Data.Tables (#14977)

pakrym committed:
Update CODEOWNERS (#14979)

christothes committed:
Reomve locale links and fix anchor links (#14987)

* Reomve locale links and fix anchor links

* fixup relative links

* move auth section under getting started

annelo-msft committed:
fb from different pr

pakrym committed:
Revert "Re-generate test common and AppConfiguration. (#14902)" (#14978)

* Revert "Re-generate test common and AppConfiguration. (#14902)"

This reverts commit 35df31a7583a0f5e68ff784a5310e12b13546eb8.

* Revert testcommon change that broke builds

MiYanni committed:
Added more information to the readme that should be generalized text across languages. Added aka.ms link. (#14990)

erich-wang committed:
Fix issue that homeAccountId string is not parsed correctly into AccountId (#13932)

conhua committed:
add .net samples (#14579)

add .net samples

ShivangiReja committed:
[Anomaly Detector] Move sample code under the right snippet (#15003)

[Anomaly Detector] Move code under the right snippet

heaths committed:
Update CHANGELOGs for Key Vault 4.2.0-beta.1 (#14992)

* Add link to sample on KeyVaultCertificate.Cer

Resolves #12742

* Update CHANGELOGs for Key Vault 4.2.0-beta.1

rajkumar-rangaraj committed:
Azure Monitor Exporter - Update codegen (#14895)

* Update codegen

* Addressing PR Comments

heaths committed:
Add Key Vault Administration package to release (#15024)

azure-sdk committed:
Update AutoRest C# version to https://github.com/Azure/autorest.csharp/releases/download/3.0.0-dev.20200909.1/autorest-csharp-v3-3.0.0-dev.20200909.1.tgz (#15026)

jsquire committed:
[Messaging Clients] Microsoft.Azure.Amqp Package Update (#15019)

The focus of these changes is to update the version of the `Microsoft.Azure.Amqp`
package used by the messaging clients to v2.4.6, which was recently
released.  This new version contains a fix for a hard crash on .NET v5
and no other changes.

azure-sdk committed:
Increment version for schemaregistry releases (#15027)

* Increment package version after release of Azure.Data.SchemaRegistry

* Increment package version after release of Microsoft.Azure.Data.SchemaRegistry.ApacheAvro

kasobol-msft committed:
[Storage] OpenWrite - support for metadata, tags and headers (#15023)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

* Ability to specify metadata tags and headers when opening write stream on block blob.

* include metadata in initial request.

asrosent committed:
EventSourceEventFormatting - catch FormatException (#14697)

* EventSourceEventFormatting - catch FormatException

* revision 2

* revision 3

* revision 4

* revision 5

* revision 6

Co-authored-by: Pavel Krymets <pavel@krymets.com>

JoshLove-msft committed:
Test Framework - Don't serialize string Json element directly to session (#15007)

Commit:
[CosmosDB] Add tableResourcesOperations scenario test (#14991)

* Add table resources operations scenario test

* Add live record file

* Update recordings

* Update recordings

* Remove extra casting

* Remove re-initialize CosmosDBClient in test file

* Add test for delete table and update the recording file

azure-sdk committed:
Increment version for keyvault releases (#15030)

* Increment package version after release of Azure.Security.KeyVault.Secrets

* Increment package version after release of Azure.Security.KeyVault.Certificates

* Increment package version after release of Azure.Security.KeyVault.Keys

* Increment package version after release of Azure.Security.KeyVault.Administration

JoshLove-msft committed:
Fix typo (#15038)

chidozieononiwu committed:
Add parameters to function (#14999)

ShivangiReja committed:
[Anomaly Detector] Fix links in the samples README (#15042)

ajaygit158 committed:
Renamed Package RegionMove to ResourceMover (#15044)

* Changed Package RegionMove to ResourceMover

* Updated resource-manager filename for regionmove to resourcemover.

Co-authored-by: Ajay Gupta <ajgupt@microsoft.com>

maririos committed:
update version to beta (#15045)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 982 (#15050)

navysingla committed:
Reoving required parameters (#15039)

DorothySun216 committed:
Enable a way to Unregister Message Handler and Session Handler (#14021)

* add UnregisterMessageHandler method

* Update sdk/servicebus/Microsoft.Azure.ServiceBus/src/Core/IReceiverClient.cs

Co-authored-by: Sean Feldman <SeanFeldman@users.noreply.github.com>

* Update sdk/servicebus/Microsoft.Azure.ServiceBus/src/Core/MessageReceiver.cs

Co-authored-by: Sean Feldman <SeanFeldman@users.noreply.github.com>

* Update the unregister method to be async and await for inflight operations to finish

* Update sdk/servicebus/Microsoft.Azure.ServiceBus/src/SubscriptionClient.cs

Co-authored-by: Sean Feldman <SeanFeldman@users.noreply.github.com>

* Update sdk/servicebus/Microsoft.Azure.ServiceBus/src/Core/MessageReceiver.cs

Co-authored-by: Sean Feldman <SeanFeldman@users.noreply.github.com>

* Update sdk/servicebus/Microsoft.Azure.ServiceBus/src/QueueClient.cs

Co-authored-by: Sean Feldman <SeanFeldman@users.noreply.github.com>

* Change name to have async suffix and add to existing onMessageQueueTests

* Add UnregisterSessionHandlerAsync and corresponding tests

* nit

* nit

* Add a new cancellation type to not cancel inflight message handling operations when unregister is called.

* Add another type of cancellation token to session handler path

* nit

* Add a timeout parameter to unregister functions and add according unit tests

* nit

* cancel runningTaskCancellationTokenSource after unregister is done

* change public API

* update the API header

* update the API definition

* fix spacing

* fix ApproveAzureServiceBus CIT test

Co-authored-by: Sean Feldman <SeanFeldman@users.noreply.github.com>

avanigupta committed:
[AppConfig] Adding new parameter 'PublicNetworkAccess' for ConfigurationStoreUpdateParameters (#14682)

* Update autorest commit ID & generate code files

* Update changelog and readme

* Fix unit test and update test recordings

* Update generated code after running eng\scripts\Export-API.ps1

ramya-rao-a committed:
Service Bus Migration Guide improvements (#14166)

sima-zhu committed:
Enable master link replacement feature in .NET (#15040)

JoshLove-msft committed:
Update changelog (#15060)

MiYanni committed:
[SchemaRegistry] Fix core build (#14968)

* Made snippet names unique.

* Fix readme links.

* Rerecorded tests and enabled them.

azure-sdk committed:
Increment package version after release of Azure.Messaging.ServiceBus (#15080)

AlexanderSher committed:
Add release date (#15084)

jsquire committed:
[Event Hubs Client] Idempotent Producer Client (#15034)

The focus of these changes is implementing the idempotent publishing feature
infrastructure into the `EventHubProducerClient` and associated types,
refactoring the current structure to support idempotent and non-idempotent
publishing as unique code paths.

Not included in these changes is the addition of the `ReadPartitionPublishingProperties`
member and live tests; those will be covered in dedicated work streams.

seanmcc-msft committed:
Fixed test recording mode (#15100)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 965 (#14909)

seanmcc-msft committed:
Added Close and RetainUncommitedData to DataLakeFileUploadOptions (#15046)

DorothySun216 committed:
Add changelog to SDK version 5.0.0 (#15073)

* add changelog to version 5.0.0

* nit

* update changelog and add release date

maririos committed:
targeting 2.1 (#15079)

azure-sdk committed:
Update AutoRest C# version to https://github.com/Azure/autorest.csharp/releases/download/3.0.0-dev.20200911.1/autorest-csharp-v3-3.0.0-dev.20200911.1.tgz (#15107)

seanmcc-msft committed:
Fixed bug where DataLakeFileClient.Rename(), DataLakePathClient.Rename(), and DataLakeDirectoryClient.Rename() couldn't handle source paths with special characters (#15106)

kasobol-msft committed:
[Storage] Make Specialized Blob Clients retrieval mockable. (#15105)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

* make specialized clients retrieval mockable.

* pr feedback.

* export api.

* record mode...

AlexanderSher committed:
Fix #14974: [BUG] DefaultAzureCredential improperly catches AuthenticationFailedException (#15057)

* Fix #14974: [BUG] DefaultAzureCredential improperly catches AuthenticationFailedExcpetion

* Check for cancellationToken

* - Update Changelog
- Add workaround for test playback

* Addressed PR comments

kerri-lee committed:
Addressed Error for Events Missing Required Fields   (#15076)

* Handled deserialization of events with missing fields

Should throw an error if required fields for EventGridEvent are missing when Parse is called

* Added tests for this error

* Added a similar constructor in CloudEvent that checks if Id (a required field) is null

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 973 (#14995)

* Sync eng/common directory with azure-sdk-tools repository for Tools PR 973

* Update update-docs-metadata.ps1

* Update update-docs-metadata.ps1

Co-authored-by: Sima Zhu <48036328+sima-zhu@users.noreply.github.com>

heaths committed:
Treat service directory as single scope (#15114)

* Treat service directory as single scope

Relates to #14969

* Resolve PR feedback

danicole committed:
Add the option to have UserAssignedMsi when authentication uses KeyVault certificate. Previously, the code would only use the system assigned MSI if any. (#14676)

Co-authored-by: dnicolescu <dnicolescu@hotmail.com>

isaacbanner committed:
Probe IMDS /metadata/instance endpoint during discovery in MsiAccessTokenProvider (#14631)

* Call imds instance endpoint instead of token endpoint to probe for imds, extend timeout

* Update msi token tests to distinguish between which imds endpoint is being called

* Fix comment and visibility in test

* Change per stpetrov

* re-trigger checks

heaths committed:
Add migration guide for Key Vault secrets (#15118)

* Add migration guide for Key Vault secrets

Relates to #12108

allenjzhang committed:
Update mgmt.yml for Azure Pipelines

Updating the casing AutoManage to Automanage

jsquire committed:
[Event Hubs Client] Idempotent Producer Client (#15125)

The focus of these changes is to enable the `EventDataBatch` to reserve
appropriate space for publishing sequence numbers, when required by the active
producer configuration and to complete the client API surface by implementing
the `ReadPartitionPublishingProperties` method.

pakrym committed:
Add EventGrid WebJobs extension (#15058)

pakrym committed:
Consume generator as a package (#15133)

Generator, Autorest shared soure and targets are all packed into a single nuget package now.

pakrym committed:
Use node 13 for CodeGeneration and merge pipelines (#14989)

stowen-msft committed:
Stowen/automanage fix casing (#15136)

* Includes Generated Files And Half Of Unit Test Coverage

* tests completed

* updated readme

* updated versions, included AssemblyInfo

* removed unneeded package references

* updated assemblyname to properCasing

* Rename Microsoft.Azure.Management.Automanage.sln to Microsoft.Azure.Management.AutoManage.sln

* casing fixes

* added metadata file

* lineending+

* newline--

* changed hardcoded reference to dotnet coreapp31

* PR feedback, fixed csproj hard coded versions, added required properties to csproj

* fixing ambiguous package name error

* fixed version number

* fix wrong reference in generate

* PRfeedback, fixed incorrect tag references, version numbers, and remove unneeded dotnet version

* pr feedback, fixing missing session records

* renamed missing test

* fixed old test

* changed casing to match RP

* bulk rename

Co-authored-by: Stephen Owen <stephen.owen@microsoft.com>

allenjzhang committed:
Update mgmt.yml for Azure Pipelines to include MySQL (#15150)

AlexGhiondea committed:
Update CODEOWNERS (#15037)

Update the AzConfig label

gechris committed:
Generate the MySQL SDKs 0.1.0 public preview (#15116)

* Add MySQL SDKs and tests

* Change to 0.1.0 version

* commit to trigger the pipelines

* updating release notes

Co-authored-by: Raj Sellappan <rajsell@microsoft.com>

gechris committed:
Generate PostgreSQL SDKs 0.9.5 public preview (#15115)

* Generate PostgreSQL SDKs and tests

* updating release notes for PostgreSQL

Co-authored-by: Raj Sellappan <rajsell@microsoft.com>

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid (#14980)

seanmcc-msft committed:
Added Close to DataLakeOpenWriteOptions, removed RetainedUncommittedData from DataLakeUploadOptions (#15140)

TylerLeonhardt committed:
Updated hybrid compute project (#14797)

* initial-updates

* working tests

* remove ctor

* new tests

* feedback

* json files

* updated session files

* update to 2020-08-02

Co-authored-by: Tyler Leonhardt (POWERSHELL) <tyleonha@microsoft.com>

heaths committed:
Add migration guides for KV certs and keys (#15147)

* Add migration guides for KV certs and secrets

Resolves #12108

* Add README links to MigrationGuide.md docs

For all services that currently have them.

azure-sdk committed:
Update AutoRest C# version (#15145)

christothes committed:
Expose select query option on GetEntity (#15161)

* Expose select query option on GetEntity

christothes committed:
Introduce an exponential retry for service timing issue causing flakey Live tests (#15159)

see https://dev.azure.com/azure-sdk/internal/_build/results?buildId=537514&view=results

sima-zhu committed:
Added skipPublishDocMs and github io for each artifact (#15119)

chidozieononiwu committed:
Update code owners file for azure template (#14767)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 999 (#15165)

JoshLove-msft committed:
Update verbiage of changelog (#15177)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 1000 (#15166)

HarveyLink committed:
Add sql test for cosmosdb (#15055)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 974 (#14997)

JoshLove-msft committed:
Update FromObjectAsync return type (#15192)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools repository for Tools PR 895 (#14996)

jsquire committed:
[Event Hubs Client] Integrate v5.3.0-beta.1 Release Branch (#15162) (#15181)

The focus of these changes is to integrate the release branch for Event Hubs
v5.3.0-beta.1.

The release introduces an option for the various event consumers allowing
the prefetch cache to be filled based on a size-based heuristic rather than
a count of events.  This feature is considered a special case, helpful in
scenarios where the size of events being read is not able to be known or
predicted upfront and limiting resource use is valued over consistent and
predictable performance.

maririos committed:
remove dev endpoint for testing (#15208)

azure-sdk committed:
Update AutoRest C# version (#15209)

christothes committed:
Reorder backup and restore operation ctor args and add sample (#15205)

fixes #15065

kasobol-msft committed:
[Storage] Queues one message receive (#15180)

* Prepare Storage for release

* pr feedback.

* PR feedback.

* pr feedback.

* make open write work with using.

* move recordings to right place

* pr feedback.

* receive one message.

* peek one message

* api

* remove redundant scope.

* pr feedback.

azure-sdk committed:
Update azure-sdk-build-tools Repository Resource Refs in Yaml files (#15216)

chidozieononiwu committed:
Organize published artifacts (#14832)

* Begin rearranging published packages

* Add step to arrange artifacts

* Remove steps for testing

* Swtich output directory

* Remove analyze and test pipeline

* Remove artifact staging from release stages

* Fix Doc Upload

* Bump Template version, remove test and analyze job

* Uncomment deployment steps

* ReAdd release stage

* Remove stage artifacts file

* Bump up template version

* Update archetype-net-release.yml and copy-docs-to-blobstorage.ps1

* Make use of VerifyPackages to retrieve pkg properties

* Update template version to test release

* Bump Azure Template

chidozieononiwu committed:
Update dotnet upload docs code (#15218)

maririos committed:
Update CHANGELOG.md (#15234)

geneh committed:
Deserialize custom dimensions from application insights API response (#14917)

* Deserialize custom dimensions from application insights API response.
Fixes #13042

* Test non-empty CustomDimensions from application insights API response

Co-authored-by: Oleg Deribas <olegd@superoffice.com>

azure-sdk committed:
Increment package version after release of Azure.AI.TextAnalytics (#15239)

azure-sdk committed:
Update AutoRest C# version (#15238)

Commit:
[CosmosDB] Add database account operations tests (#15189)

heaths committed:
Fix error in directions in Key Vault READMEs (#15213)

weshaggard committed:
Switch ApiCompat to use PackageDownload (#15250)

Turns out instead of using the apicompat project workaround
we can use a PackageDownload reference to get the last shipped
package. This allows us to do the apicompat check completely
inside the project we are checking.

sima-zhu committed:
Replaced relative link with absolute links and remove locale (#15223)

Replaced relative link with absolute links and remove locale

Commit:
Add MongoDB operations tests and it's recordings (#15254)

sima-zhu committed:
Enable the link check on aggregate-report (#15257)

serkantkaraca committed:
Increase diagnostics-id overhead in EventDataBatch (#15193)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.ServiceBus (#15111)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.HDInsight.Job (#14291)

xioyi committed:
Update EventTypes.cs (#13569)

heaths committed:
Fix nullability on Search models (#15240)

Fixes #15108

rajkumar-rangaraj committed:
Update CODEOWNERS (#15244)

* Added azure monitor entry.

* Change label entry

* New label

azure-sdk committed:
Increment version for synapse releases (#14751)

* Increment package version after release of Azure.Analytics.Synapse.Spark

* Increment package version after release of Azure.Analytics.Synapse.Artifacts

* Increment package version after release of Azure.Analytics.Synapse.AccessControl

azure-sdk committed:
Increment version for identity releases (#15113)

Increment package version after release of Azure.Identity

jsquire committed:
[Azure Messaging] README Updates (Troubleshooting + Logging) (#15237)

The focus of these changes is to enhance the troubleshooting section of
the README documentation with information about logging and distributed
tracing, calling out to the existing Azure.Core resource for details.

abatishchev committed:
Fixed link to Azure SDK Design Guidelines for .NET (#15210)

pakrym committed:
Increase connection pool size limit automatically (#15263)

Kotasudhakarreddy committed:
updated .NetSDK version for Websites to 2020-06-01 (#15230)

* updated .NetSDK version for Websites to 2020-06-01

* Updated API version targetting to 2020-06-01

Commit:
Fix potential deadlock in checking database account name existence (#15255)

bquantump committed:
Dns ZoneType Change (#15236)

* WIP: Initial CRUD and List test for record and zone

* WIP: Port of cloud issues

* WIP:  OneTimeTearDown and setup working. No double test runs

* WIP: don't make zone twice

* WIP: Commit test

* WIP: Fix changelog and readme

* WIP: Fix yaml

* WIP: Fix yml 2

* WIP: Remove unused function

* Remove .swp file

* Remove SWP file

* Update DNS Track 2 test

* Add Tests Recordings

* Fix Class name

* WIP: Update tests

* WIP: Make exception test clearner

* WIP: Update live test records

* Revert json change

* Small fixes

* WIP: Update SLN

* WIP: Dns gen update

* Revert "WIP: Dns gen update"

This reverts commit 2c0317c8aa3a2629eb85818ec1475be1ce8bae39.

* WIP: make zonetype obsolete

* WIP: make zone type readonly

* WIP: update test

* WIP: Make all private relate fields obsolete

* WIP: only change things we want to change

* WIP: Update test

* WIP: Update test

* WIP: update test correctly

Co-authored-by: bqauntump <bqauntump@DESKTOP-B63NOF4.localdomain>

heaths committed:
Remove obsolete conditions from Search (#15268)

Fixes #14096 to simplify the code for upcoming changes.

annelo-msft committed:
Merge remote-tracking branch 'upstream/master' into api-learn-api-activity

annelo-msft committed:
pr fb

jsquire commented:
Hi @annelo-msft.   Can you help me understand if this work is still active or we should close out the PR?

annelo-msft committed:
Revert "Merge remote-tracking branch 'upstream/master' into api-learn-api-activity"

This reverts commit e4833531342f10a7fcf2bfc038a658cbe174d7d4, reversing
changes made to 8b7b2d66affb25c38a0bd506c553fd6b774969b9.
